src/entities/drawer/lib/utils/LineSketcher.ts

```diff
@@ -0,0 +1,92 @@
+import { MutableRefObject } from 'react';
```

Comment:
Please ensure:
- Bezier working the same as in the previous version
- Dots
- Lines sometimes are'not started drawing, details in the Teams